src/Mono.Data.Sqlite/Mono.Data.Sqlite.csproj

```diff
@@ -11,10 +11,8 @@
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <AndroidResgenClass>Resource</AndroidResgenClass>
     <AndroidResgenFile>Resources\Resource.designer.cs</AndroidResgenFile>
-    <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
```

Comment:
What IDE version are you using that it's making these changes? XS 6.0 seems to be using `True` instead of `true` now, so I'm hesitant/curious about these `<PropertyGroup/>` changes.

The `@(Reference)` changes should be fine.